src/Components/MonthPicker.js

```diff
@@ -0,0 +1,55 @@
+import DropDownPicker from 'react-native-dropdown-picker';
+import React, { useState } from 'react';
+import { monthName } from '../functions/monthName';
+
+export const MonthPicker = ({setTime}) => {
+    const currentMonth = new Date().getMonth() + 1; 
+    var month3;
+    const currentYear = new Date().getFullYear();;
+    switch(currentMonth) {
+        case 1:
+            month3 = 10;
+            break;
+        case 2:
+            month3 = 11;
+            break;
+        case 3:
+            month3 = 12;
+            break;
+        default:
+            month3 = currentMonth - 3;
+    }
+
+    const [open, setOpen] = useState(false);
+    const [items, setItems] = useState([
+        {label: 'Month', value: 'Month'},
+        {label: `${monthName(month3)}`, value: `${month3}`, parent: 'Month'},
+        {label: `${monthName(month3 + 1 % 12)}`, value: `${month3 + 1 % 12}`, parent: 'Month'},
+        {label: `${monthName(month3 + 2 % 12)}`, value: `${month3 + 2 % 12}`, parent: 'Month'},
+        {label: `${monthName(currentMonth)}`, value: `${currentMonth}`, parent: 'Month'},
+
+        {label: 'Year', value: 'Year'},
```

Comment:
Just to check that I understood this correctly : we allow the user to choose any of the previous 3 months to view their expenses

Reply:
That's right! I included the previous year as well for the case of any of the previous 3 months being in the preceding year